specification/glossary.md

```diff
@@ -38,7 +38,7 @@ A row in a FOCUS-compatible cost and usage dataset.
 
 <a name="glossary:chargeperiod"><b>Charge Period</b></a>
 
-The time window in which a charge was incurred, inclusive of the start date and exclusive of the end date. A charge can start and/or end at any time within a charge period window. The charge period for continuous usage should match the time granularity of the dataset (e.g., 1 hour for hourly, 1 day for daily).
+The time window for which a charge is effective, inclusive of the start date and exclusive of the end date. The charge period for continuous usage should match the time granularity of the dataset (e.g., 1 hour for hourly, 1 day for daily). The charge period for a non-usage charge with time boundaries should match the duration of eligibility.
```

Comment:
Left this purchase agnostic to account for charges such as: 
credits that expire 1 year from now are given but not consumed.

Reply:
@aqu-erp, Andrew, can you please resolve this comment? The group has agreed to the PR, and it can be merged.